tests/bluetooth/tester/src/audio/btp_bap_broadcast.c

```diff
@@ -125,15 +125,30 @@ static void stream_started(struct bt_bap_stream *stream)
 	btp_send_bis_syced_ev(&broadcaster->address, broadcaster->broadcast_id, b_stream->bis_id);
 }
 
+static void btp_send_bis_stopped_ev(const bt_addr_le_t *address, uint32_t broadcast_id,
+				  uint8_t bis_id, uint8_t reason)
```

Comment:
```suggestion
static void btp_send_bis_stopped_ev(const bt_addr_le_t *address, uint32_t broadcast_id,
				    uint8_t bis_id, uint8_t reason)
```

Reply:
Right, my editor uses a different standard indentation, but updated to look nice on git now.

Reply:
If your editor supports https://editorconfig.org/ then it's easy to ensure that it works with many projects out of the box :)